docs/requirements.txt

```diff
@@ -49,6 +49,8 @@ rasterio>=1.0.16
 rtree>=0.5
 # scikit-learn 0.18+ required for sklearn.model_selection module
 scikit-learn>=0.18
+# scipy 1.5.4+ required for scipy.io.wavfile.read
```

Comment:
This function existed long before 1.5.4 (https://docs.scipy.org/doc/scipy-0.9.0/reference/generated/scipy.io.wavfile.read.html). I don't even see any mention of this function in the 1.5.4 changelog. Does it really not work with any older version of scipy? What changed in 1.5.4?

EDIT: Just noticed:

> Retrying with scipy>=1.5.4 since the last release for ubuntu python 3.6 is 1.5.4.

What's the first version of scipy that supports Python 3.6? If it's greater than 0.9.0 (when the function was added) we can use that as the minimum required version.

Reply:
Running in a google colab with Python 3.6, it seems that 0.9.0 breaks when installing. I'm indifferent on whether make the minimum required version as 0.9.0 but it seems as though this will likely not work for all the Python version we are checking.

Reply:
To clarify, it's not the function itself that changed, but previous versions of scipy appear to not be installable due to various issues.

Reply:
I would just make it 0.9.0, it seems like Python 3 was supported back then. If someone is on an older system they might not be able to install newer versions of scipy, so we don't want to artificially impose a newer requirement than we actually have.

Reply:
Updated to 0.9.0